Lint and format JS files

npalladium opened the pull request:
- Run ESLint to fix a bunch of issues
- Run Prettier with default config
- Fix a snake_case to camelCase

Don't know if code works though.

npalladium committed:
Lint and format JS files

- Run ESLint to fix a bunch of issues
- Run Prettier with default config
- Fix a snake_case to camelCase

Don't know if code works though.

RachitKeertiDas commented:
Thanks for this, @npalladium !

Good catch for the snake_case variables.

Code works and changes LGTM. 

**P.S.** : As discussed, since it's mostly indentation changes, I'll follow up this with a commit for `git-blame-ignore-revs`  and enforcing strict mode.

npalladium commented:
A few other things to do: 
- Switch to modern JS
- Possibly commit the linting/prettier configs.

RachitKeertiDas reviewed (approved):
LGTM.

RachitKeertiDas committed:
Fix Most Lint Errors. Commit Lint Config Files.

Add Prettierrc and ESLintrc  by @npalladium.
Use Strict Mode
Fix Possible Bug in popup.js where error message may not be displayed.
Remove unnecessary console.log()
